### DISCUSSION TITLE
MOM6 PanAntarctic @ 0.05°

### DISCUSSION THREAD

**Discussion by AndyHoggANU** [COSIMA/mom6-panan#5] [General]
This discussion is to document what we have learn from running a series of MOM6 PanAntarctic runs.

1. Early tests focussed on getting the open boundary conditions working effectively. The key aspect here was to get the tracer reservoirs to be the right size (although we have limited understanding of how that actually works).
2. Our first full run with the z* coordinate identified some high CFL regions in the Ross Sea that ultimately limits our timestep. We marginally improved this by smoothing the topography.
3. We did some testing on scalability, and opted for 3715 cores (actually 3744) ...
4. We did a long, 8-year run with the z* coordinate -- called `panant-v2`.  Also did a run called `panant-hycom1` which went for 5 years and used the MOM6 hybrid grid approach. These runs were OK, but increased DSW formation in this configuration doesn't interact so well with the northern boundary flux constraints.
5. Did two more runs, which went for three years: `panant-v3` and `panant-hycom1-v2`. These had temperature and salinity sponges on the northern boundary to help close the enhanced overturning circulation. This was effective, but killed the upper overturning and also induced some eddies along the northern boundary. Concluded that this is not the way to proceed.
6. Note that the latest runs also had increased resolution near the bottom for dense water overflows, which is more appropriate.
7. Meanwhile, we seem to have discovered that the above runs all use an interpolated version of river runoff. Likely, you can't interpolate river runoff ... in fact we only get 10-20% of the runoff that we should have. This needs to be fixed for the next version.

Based on what we have learnt here, we will now do the following:
* Restart all runs, after interpolating the river runoff onto the model grid.
* Include more diagnostics in the output, retaining new diagnostic axes.
* Run at 0.1° for now, so we can better compare with ACCESS-OM2 -- and will run quicker.
* Turn the sponging code back off.

**Comment by adele-morrison** [comment]
Nice, thanks for keeping the hard work going here! It seems plausible that increasing the runoff by 5-10 times will reduce the DSW formation sufficiently that there won't be the same overturning problems as before.
